recommenders/popularity_recommender.py

```diff
@@ -1,31 +1,27 @@
-from recommenders import data_movies, data_ratings, default_poster_url
-import pandas as pd
-
-
-def recommend_popular_movies():
-
-    ######################################################################################################
-    movie_ratings = None
```

Comment:
The variable 'movie_ratings' is initialized to None on line 8 but redefined on line 11. Consider removing the initial assignment on line 8 since it is not necessary.